3_javascript/javascript3/week2/script.js

```diff
@@ -0,0 +1,68 @@
+
+const ApiKey = `https://open.er-api.com/v6/latest/EUR`;
+
+
+
+let toSelectList = document.createElement("select");
+let fromSelectList = document.createElement("select");
+let rates;
+
+fetch(ApiKey)
+    .then(Response => Response.json())
+    .then(data => { //funA
+        rates = data.rates;
+        changeCurrency(data.rates)
+    });
+
+function changeCurrency(rates) {//funcB
+    //select list from 
+
+    const fromCurrency = document.getElementById("fromCurrencyContainer")
+    fromSelectList.id = "SelectFrom";
+    fromCurrency.appendChild(fromSelectList);
+
+    let key = Object.keys(rates)
+
+    for (var i = 0; i < key.length; i++) {
```

Comment:
In modern JavaScript `var` is not used anymore. See https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/